### PR TITLE
Adds script and instructions for usage in docker

### PR DESCRIPTION
**Pull request by mhbuehler**
This PR adds a script and multi-stage dockerfile to `tools` and instructions to the main `README.md` for installing and running ngraph-bridge in docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,65 @@ Note: The version of the ngraph-tensorflow-bridge is not going to be exactly
 the same as when you build from source. This is due to delay in the source 
 release and publishing the corresponding Python wheel. 
 
+### Build and run nGraph in Docker
```
Comment (shresthamalik) [inline README.md]: Can we move this to another README for docker build. You can have a link to that in this README

File: tools/Dockerfile.ubuntu18.04.install [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ARG https_proxy
+ARG base_image
+ARG ngtf_build_options
+
+FROM $base_image as base
+ARG http_proxy
+ARG https_proxy
+ARG ngtf_build_options
+
+RUN git clone https://github.com/tensorflow/ngraph-bridge.git /src/ngraph-bridge
```
Comment (sindhu-nervana) [inline tools/Dockerfile.ubuntu18.04.install]: we don't have `src` anymore. it will be just `ngraph-bridge/ngraph_bridge`

File: README.md (hunk 1 above)
Comment (mhbuehler) [reply README.md]: 👍 I've done this in PR #355 . Please switch over to that PR.

File: tools/Dockerfile.ubuntu18.04.install (hunk 2 above)
Comment (mhbuehler) [reply tools/Dockerfile.ubuntu18.04.install]: I removed the `/src`prefix in PR #355 . Please switch over to that PR which resolves the CLA problem.

Comment (sindhu-nervana) [reply tools/Dockerfile.ubuntu18.04.install]: So, is this PR a duplicate? Can we close this?

**Comment by ashahba** [comment]
@mhbuehler and @kkasravi can we get this one closed now since #355 is merged?

**Pull request closed without merging**